### ISSUE TITLE
Chestxray14 labeled subset setup

### ISSUE THREAD

**Issue by ThomasLiu1021** [liuquande/SRC-MT#3]
Hi,
I am curious about the multi-label chestxray14 experiment setup. Because this dataset contains serious data imbalance problems, I wonder is your 20% labeled dataset a balanced subset or you just random sampling them? 

Looking forward to your reply.

Cheers

**Comment by liuquande** [comment]
We random sample them and keep the the ratio of each class in the validation set the same as the original training set.

**Comment by ThomasLiu1021** [comment]
Thanks for the clarification.  I am closing the issue.
